paparazzi/settings.gradle

```diff
@@ -1,4 +1,13 @@
-rootProject.name = 'paparazzi-libs'
+/**
+ * Set the root project name to `paparazzi` when building from Intellij or Android Studio so that run executions created
+ * via context actions resolve to the task that matches the actual project directory structure. This is a workaround for
+ * https://github.com/gradle/gradle/issues/16608.
+ */
+if (System.getProperty("idea.active")) {
```

Comment:
Is this a documented prop? Where did you find it? (Just curious, because I can't seem to find good description)